codec-http2/src/test/java/io/netty/handler/codec/http2/Http2ConnectionRoundtripTest.java

```diff
@@ -1272,6 +1274,11 @@ private static Http2Headers dummyHeaders() {
         .add(randomString(), randomString());
     }
 
+    private static Http2Headers dummyTrailers() {
+        return new DefaultHttp2Headers(false)
+        .add(randomString(), randomString());
```

Comment:
```suggestion
        .add("header-" + randomString(), randomString());
```
This will result in some slight flakyness because we can randomly generate a random header that starts with `:`.

Reply:
Oh I didn't think about that, will incorporate. Thanks!!